aloha-vw-jni/src/test/scala/com/eharmony/aloha/models/vw/jni/VwJniModelTest.scala

```diff
@@ -79,29 +81,31 @@ class VwJniModelTest extends Logging {
     }
 
 
-    @Test def testFailureWhenVwIsCreatedWithLinkParamAndInitialRegressorHasSameLink(): Unit = {
-        try {
-            val src = Base64EncodedBinaryVwModelSource(logisticModelB64Encoded, LogisticModelParams)
-            src.vwModel(1).close()
-            fail("VW should throw a java.lang.Exception with message: \"option '--link' cannot be specified more than once\"");
-        }
-        catch {
-            // Can't rely on msg because different versions of boost (a dep of VW) return different messages
-            // case e: IllegalArgumentException if e.getClass.getSimpleName == "IllegalArgumentException" && e.getMessage == "option '--link' cannot be specified more than once" =>
-            case e: IllegalArgumentException if e.getClass.getSimpleName == "IllegalArgumentException" =>
-            case t: Throwable => throw t
-        }
+    @Ignore @Test def testFailureWhenVwIsCreatedWithLinkParamAndInitialRegressorHasSameLink(): Unit = {
+//        try {
```

Comment:
Whoops.  Those were commented out because they don't really belong here.  Those should go in vw jni library.